lib/rtags.js

```diff
@@ -174,7 +174,7 @@ module.exports.RcExecutor = class RcExecutor extends EventEmitter {
           reject(stdout);
         }
         this.rdmStarted = true;
-        this.emit('rdmStarted')
+        this.emit('rdmStarted');
```

Comment:
Since this is a bug fix for the earlier commit, you should squash these into one commit before sending a pull-request.  It keeps the project cleaner.

Reply:
Finally got time to do it...
By the way do you have any clue why the CI is failing (seems unrelated to my change)